src/screens/MainApp/NoPhotoDisclaimer/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { View, Image } from 'react-native'
+import { Text, Button } from '~/components'
+import DrCannabis from '~/assets/images/DrCannabis.png'
+import styles from './styles'
+
+const goToCallToAction = () => {
+  console.log('Going to Call To Action')
+}
+
+const NoPhotoDisclaimer = () => (
+  <View style={styles.container}>
+    <Image source={DrCannabis} style={styles.image} />
+    <Text colorVariant='gray'>
+      Ya no tenés fotos para enviar
```

Comment:
Not sure how React handles accents. Shouldn't we be using html entities instead?

cc @nahudalla

Reply:
I think it should handle any valid JavaScript string here, since this is transpiled into a string by Babel. JavaScript strings are UTF-16, so it should work correctly in my opinion.